ietf/doc/views_ballot.py

```diff
@@ -12,11 +12,11 @@
 from django.shortcuts import render, get_object_or_404, redirect
 from django.template.defaultfilters import striptags
 from django.template.loader import render_to_string
-from django.urls import reverse as urlreverse, resolve as urlresolve, Resolver404
+from django.urls import reverse as urlreverse
 from django.views.decorators.csrf import csrf_exempt
 from django.utils.html import escape
 from urllib.parse import urlencode as urllib_urlencode
-
+from .return_to_url import parse_ballot_edit_return_point
```

Comment:
Minor, but organizationally we generally split imports into "basic packages", "django stuff", and "datatracker stuff" (i.e., `ietf.*`), roughly alphabetically in each section. This should then move down into the `ietf.*` section.

We're also slowly adopting the relative import syntax, so great to use it. If it's in a module that's not already using it, I've usually just followed whatever style is used there, but we should probably convert existing imports from `ietf.thisapp.whatever` to `.whatever` when doing so to keep it consistent. 

(If you accept my reorg suggestions then that'll be moot here, though)